updates/control.sh

```diff
@@ -234,6 +234,20 @@ run_hooks() {
     done
 }
 
+renew_dhcp_after_hwinstalling () {
+    # The Admin IP address of the node is allocated in the
+    # "hardware-installing" transition. Force a renewal here get the new IP
+    # address and avoid it changing and an unexpected time during the
+    # subsequent transitions
+    if [[ $1 = 'hardware-installing' ]]; then
+        echo "Forcing DHCP renewal after Admin IP allocation"
+        ifup $BOOTDEV > /dev/null
```

Comment:
You mean distros != SUSE? I have no idea. But it should work on all SUSE distros.